src/xcode/ENA/ENA/Source/Workers/Store/Store.swift

```diff
@@ -157,6 +157,8 @@ protocol PrivacyPreservingProviding: AnyObject {
 	var previousRiskExposureMetadata: RiskExposureMetadata? { get set }
 	/// Analytics data.
 	var userMetadata: UserMetadata? { get set }
+    /// Analytics data.
```

Comment:
Maybe this is just Github, but
```suggestion
     /// Analytics data.
```